aikau/src/main/resources/alfresco/navigation/_HtmlAnchorMixin.js

```diff
@@ -42,7 +42,7 @@ define(["dojo/_base/declare",
         "dojo/query",
         "dojo/NodeList",
         "dojo/NodeList-manipulate"], 
-        function(declare, AlfCore, _NavigationServiceTopicMixin, AlfConstants, lang, array, query, NodeList, NodeListManipulate) {
+        function(declare, AlfCore, _NavigationServiceTopicMixin, AlfConstants, lang, array, query /*, NodeList, NodeListManipulate*/) {
```

Comment:
The reason is that the dependencies aren't required, but they aren't referenced. I could have JSHinted them out, but I wanted to make sure that if someone adds in additional dependencies that they don't end up with a mis-match between the array of dependencies and the callback args